release.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+get_latest() {
+  git fetch
+
+  git tag |
+    # drop `v` prefix
+    grep "^v" |
+    cut -c2- |
+
+    # sort by semantic version
+    sort -t "." -k1,1n -k2,2n -k3,3n |
+
+    # last
+    tail -n 1 |
+
+    # drop newline
+    tr -d '\n'
+}
+
+NEW="$1"
+
+if [ -z "$NEW" ]; then
+  echo "Usage  : bash release.sh <version>"
+  echo "Example: bash release.sh 1.2.3"
+  echo ""
+  echo "Fetching tags..."
+  latest="$(get_latest)"
+  echo -n "The current version is: $latest"
+
+  exit 1
+fi
+
+if [[ "$NEW" == v* ]]; then
+  echo "<version> must not start with \"v\""
+  exit 1
+fi
+
+echo "Checking for clean working tree..."
+if [[ "$(git diff --stat)" != "" ]]; then
+  echo "❌ Dirty working tree (try git stash)"
+  exit 1
+fi
+
+echo "Checking that we're on master..."
+if [[ "$(git symbolic-ref HEAD | tr -d '\n')" != "refs/heads/master" ]]; then
+  echo "❌ Not on master (try git checkout master)"
+  exit 1
+fi
+
+echo "Checking that master is up to date..."
+git fetch
+if [[ "$(git rev-parse master)" != "$(git rev-parse origin/master)" ]]; then
+  echo "❌ master is out of sync with origin/master (try git pull)"
+  exit 1
+fi
+
+git ls-tree -r HEAD --name-only -z examples | xargs -0 sed -i.sedbak "s/\"[0-9]*\.[0-9]*\.[0-9]*\" # LATEST/\"$NEW\" # LATEST/g"
```

Comment:
Does this only replace things that are suffixed with `# LATEST`? It would be nice if we could ensure that there are links to the right tag when targetting the repo in the .md files as well.

Reply:
:+1: I'll change this and `.buildkite/check-latest-tag.sh` in a separate PR